### PR TITLE
kodi: update to kodi-734cda1 (latest kodi 18a1 master)

### PR DESCRIPTION
**Pull request by MilhouseVH**
Brings Kodi up to date.

Bumps ffmpeg, libiconv, libfmt and libmicrohttpd to remain in sync with Kodi.

### CODE REVIEW

File: packages/mediacenter/kodi/config/appliance.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8" ?>
-<settings>
+<settings version="1">
```
Comment (lrusak) [inline packages/mediacenter/kodi/config/appliance.xml]: intended?

**Comment by lrusak** [comment]
looks ok. kodi patches are still ugly. :+1:

File: packages/mediacenter/kodi/config/appliance.xml (hunk 1 above)
Comment (MilhouseVH) [inline packages/mediacenter/kodi/config/appliance.xml]: Yes, you now get a warning without it:
```
18:09:55.691 T:140403180255296   DEBUG: CSettings: loaded settings definition from special://xbmc/system/settings/appliance.xml
18:09:55.691 T:140403180255296 WARNING: CSettingsManager: missing version attribute
```

**Comment by InuSasha** [comment]
packages/multimedia/ffmpeg/patches/ffmpeg-99.1001-pfcd_hevc_optimisations.patch does not apply, after this commit

**Comment by MilhouseVH** [comment]
Yep my bad, I pull that in automatically from popcornmix's newclock5 branch - I'll push the ffmpeg update in a new PR.

**Comment by MilhouseVH** [comment]
@InuSasha #1760 should fix it, sorry about that!

**Comment by InuSasha** [comment]
np, can happen. (there must be a reason for have same tester ;) )
Now it builds :)

**Pull request merged**